package/MDAnalysis/lib/mdamath.py

```diff
@@ -36,18 +36,27 @@
 
 from ..exceptions import NoDataError
 
+
 # geometric functions
 def norm(v):
-    r"""Returns the length of a vector, ``sqrt(v.v)``.
```

Comment:
In general, you should use raw strings when you use _any_ LaTeX markup. As soon as you write ``\rho`` or ``\nu`` you'll get in trouble because the control sequences are suddenly expanded.